docs/content/latest/architecture/transactions/distributed-txns.md

```diff
@@ -96,32 +95,40 @@ the one-byte prefix that puts these records before all regular records in RocksD
     explicitly passed to all components handling a particular transaction.
   - `Isolation Level` [Snapshot Isolation](https://en.wikipedia.org/wiki/Snapshot_isolation) or
     [Serializable Isolation](https://en.wikipedia.org/wiki/Serializability).
-  - `Priority` This priority is assigned randomly during transaction creation. When a conflict
-    is detected between two transactions, the transaction with lower priority is
-    aborted and restarted.
+  - `Priority` This priority is assigned randomly during transaction creation when optimistic
+    concurrency control is used. For transaction that is being run under pessimistic concurrency
+    control this priority is assigned to a very high value. When a conflict is detected between two
+    transactions, the transaction with lower priority is aborted and restarted.
+    See [explicit locking in YugabyteDB](../explicit-locking) for more details about concurrency
+    control.
 
-#### 3. Provisional record keys indexed by transaction ID** ("reverse index")
+#### 3. Provisional record keys indexed by transaction ID ("reverse index")
 
 ```
 TxnId, HybridTime -> primary provisional record key
 ```
 
-  This mapping allows us to find all RocksDB records belonging to a particular transaction.  This is
-  being used when cleaning up committed or aborted transactions. Note that because multiple RocksDB
-  key-value pairs belonging to primary provisional records can we written for the same transaction
-  with the same hybrid time, we need to use an increasing counter (which we call a *write id*) at
-  the end of the encoded representation of hybrid time in order to obtain unique RocksDB keys for
-  this reverse index. This write ID is shown as `.0`, `.1`, etc. in `T130.0`, `T130.1` in the figure
-  above.
+  This mapping allows us to find all provisional RocksDB records belonging to a particular
+  transaction. This is being used when cleaning up committed or aborted transactions. Note that
+  because multiple RocksDB key-value pairs belonging to primary provisional records can we written
+  for the same transaction with the same hybrid time, we need to use an increasing counter (which we
```

Comment:
Not sure what others said about "hybrid time" versus "hybrid timestamp", but I think this should be "hybrid timestamp" because that is what gets written to the transaction.